src/app/docs/resources/faq/page.mdx

```diff
@@ -8,7 +8,7 @@ description: "Find answers to frequently asked questions about kagent."
 
 ## What's the best way to get started with kagent?
 
-The best way to get started with kagent is to follow the [quickstart guide](/docs/getting-started/quickstart). This will give you a basic understanding of how kagent works and how to use it.
+The best way to get started with kagent is to follow the [quickstart guide](https://kagent.dev/docs/getting-started/quickstart). This will give you a basic understanding of how kagent works and how to use it.
```

Comment:
```suggestion
The best way to get started with kagent is to follow the [quickstart guide](/docs/getting-started/quickstart). This will give you a basic understanding of how kagent works and how to use it.
```